### PR TITLE
LPS-197681 - Prevent users to save fields multiple times

### PR DESCRIPTION
**Pull request by juanjofgliferay**
Related bug https://liferay.atlassian.net/browse/LPS-197681

## Problem
It is possible to add fields more than once in a Dataset View

## Proposed solution
Disabled **Save** button while making the request

### Before 

[Screencast from 2023-09-29 09-14-03.webm](https://github.com/liferay-frontend/liferay-portal/assets/132653342/ae037f47-b513-4dc7-bc34-4d75435ff15e)

### After

[Screencast from 2023-09-29 10-03-53.webm](https://github.com/liferay-frontend/liferay-portal/assets/132653342/97b1a4b4-6aa9-43b6-ab24-9262fa3ea629)

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx [hunk 1]
```diff
@@ -158,6 +161,7 @@ const SaveFDSFieldsModalContent = ({
 	};
 
 	const saveFDSFields = async () => {
+		setSaveButtonDisabled(true);
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx:164]: [nit] We have a common convention to add an empty line of code between two unrelated sibling lines of code. As an example of what is related in this file, multiple state declarations are bundled together, and multiple variable declarations. This is not a hard rule, it's not enforced by source formatter. It is kinda difficult to write a rule for this, as it is difficult to define what is 'related'. Nevertheless, we do have such set of rules for Java and JSPs, but not in JS/TS. Here, it is only a non-blocking convention.

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! A non-blocking not in comment.

**Review by dsanz — APPROVED** [review]

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx (hunk 1 above)
Comment (juanjofgliferay) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx:164]: Thanks for caring about those little details. I usually try to keep this extra space, but without an automation is not always easy.

**Comment by juanjofgliferay** [comment]
ci:forward

**Comment by juanjofgliferay** [comment]
ci:forward

**Comment by JerryAdrianoo** [comment]
Hey, @juanjofgliferay these failures are not related to your changes!!!!
[Test Comparison](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1696012667227&testrayRunIdA=2433035916&testrayRunIdB=2433955812)

**Comment by thektan** [comment]
Manually forwarded this to BChan https://github.com/brianchandotcom/liferay-portal/pull/141580

**Comment by thektan** [comment]
ci:close

**Pull request closed without merging**